cmd/lotus-miner/actor.go

```diff
@@ -40,6 +40,7 @@ var actorCmd = &cli.Command{
 	Subcommands: []*cli.Command{
 		actorSetAddrsCmd,
 		actorWithdrawCmd,
```

Comment:
Can we rename this to ownerWithdrawCmd, rename the command itself to `owner-withdraw`, but leave `withdraw` as an alias?

Reply:
Refactored everything. Now the cli commands just need a flag to know to send from beneficiary.